.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,22 @@
+name: Code Quality (gofmt, gotest)
+on: [pull_request, create]
+
+jobs:
+  build:
+    if: github.event_name == 'pull_request'
+    name: Code Quality (gofmt, gotest)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
+      
+      - name: Formatting (goimports)
```

Comment:
This is what failed the test in actions. Should just need the dependency resolved locally.

Reply:
also planning to switch to `make format.check` so it will fail on unformatted code.

Reply:
switched to `make format.check` and added `PATH=${PATH}:`go env GOPATH`/bin` to find goimports.